File: src/main/java/com/github/jactor/persistence/repository/BlogEntryRepository.java
package com.github.jactor.persistence.repository;

import com.github.jactor.persistence.entity.BlogEntryEntity;
import org.springframework.data.repository.CrudRepository;

import java.util.List;

public interface BlogEntryRepository extends CrudRepository<BlogEntryEntity, Long> {
    List<BlogEntryEntity> findByBlog_Id(Long blogId); // the underscore in determined by spring jpa
}

File: src/main/kotlin/com/github/jactor/persistence/repository/BlogEntryRepository.kt
package com.github.jactor.persistence.repository

import com.github.jactor.persistence.entity.BlogEntryEntity
import org.springframework.data.repository.CrudRepository

interface BlogEntryRepository : CrudRepository<BlogEntryEntity?, Long?> {
    fun findByBlog_Id(blogId: Long?): List<BlogEntryEntity?>? // the underscore in determined by spring jpa
}
